tests/common/GenerateStringResponses.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#include "GenerateStringResponses.h"
+
+#include <string>
+#include <vector>
+
+#include <olp/dataservice/read/model/Partitions.h>
+#include <olp/dataservice/read/model/VersionResponse.h>
+#include "ApiDefaultResponses.h"
+#include "ReadDefaultResponses.h"
+#include "UrlGenerators.h"
+#include "generated/model/Api.h"
+// clang-format off
+#include "generated/serializer/ApiSerializer.h"
+#include "generated/serializer/VersionResponseSerializer.h"
+#include "generated/serializer/PartitionsSerializer.h"
+#include "generated/serializer/JsonSerializer.h"
+// clang-format on
+
+namespace {
+template <class T>
+std::string serialize(std::vector<T> data) {
+  std::string str = "[";
+  for (const auto& el : data) {
+    str.append(olp::serializer::serialize(el));
+    str.append(",");
+  }
+  str[str.length() - 1] = ']';
+  return str;
+}
+}  // namespace
+
+std::string ResponseGenerator::ResourceApis(const std::string& catalog) {
+  auto api_response =
+      mockserver::ApiDefaultResponses::GenerateResourceApisResponse(catalog);
+  return serialize(api_response);
+}
```

Comment:
Why we need custom serialize here?
We have serialize for vector and for api, so should be no problem to use`olp::serializer::serialize(api_response)`